manifests/init.pp

```diff
@@ -112,6 +112,10 @@
     warning('data_dir must be set to install consul web ui')
   }
 
+  if ($config_hash_real['ports'] and $config_hash_real['ports']['rpc']) {
+    $custom_rpc_port = $config_hash_real['ports']['rpc']
+  }
```

Comment:
it a good practice to initialize all variables, even if you have to do `undef`. In other words it should be like:

```
} else {
  $custom_rpc_port = 8400
}
```

Also, can we just name this `rpc_port`?